### PR TITLE
Updated CHANGELOG and package.json for 2.15.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 2.15.0 (2019-06-06)
+
+- Updated omnibar placeholder accent to match the new omnibar gradient accent. (#92)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I'd add "the" after "Updated." ... Also, in most of our changelogs, the number at the end links to the PR. Is it worth doing that here as well?

Comment (Blackbaud-BobbyEarl) [reply CHANGELOG.md]: For whatever reason, we've not linked the PR numbers in this repo.  I would say leave it as it for consistency, unless we're going to link all of them.

Comment (blackbaud-johnly) [reply CHANGELOG.md]: Oh, I agree about doing all or none. Sorry if that wasn't clear. ... I can make the change if we are inclined to add links, but I need rights to edit the changelog.

Comment (Blackbaud-PaulCrowder) [reply CHANGELOG.md]: I updated all the entries in the changelog to include PR links.

**Pull request merged**